src/definitions/DungeonRaidInterfaces.ts

```diff
@@ -644,7 +658,19 @@ export interface IRaidInfo {
      *
      * @type {string}
      */
-    vcId: string;
+    vcId: string | null;
+
+    /**
+     * The raid ID
+     */
+    raidId: string;
+
+    /**
+     * Whether the raid is vcless
+     *
+     * @type {boolean}
+     */
+    vcless: boolean;
```

Comment:
Hell no.  I have had to fight your dependence on the VC existing as a metric for determining various states of existence for the instances.  It's a terrible design decision and you know it.  Vcless is a valuable trait to keep track of for a  raid.

Reply:
It's probably terrible because VCless wasn't something that I thought would need to be added. 

If it's such a problem, you're free to fix it — as long as it doesn't involve copying the code and then making minor changes.

Reply:
I did fix it, by adding this boolean